src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -655,6 +655,53 @@ std::vector<Feature> Renderer::Impl::queryRenderedFeatures(const ScreenLineStrin
 
     return result;
 }
+    
+std::vector<Feature> Renderer::Impl::queryShapeAnnotations(const ScreenLineString& geometry) const {
+    std::vector<const RenderLayer*> layers;
+    RenderedQueryOptions options;
+    options.layerIDs = {{ AnnotationManager::ShapeLayerID }};
```

Comment:
There isn’t ever a layer with this ID, is there? Does mbgl use `options.layerIDs` for anything past this point?